src/core/public/styles/_ace_overrides.scss

```diff
@@ -12,6 +12,8 @@
 
     background-color: $euiColorLightestShade;
     color: $euiTextColor;
+    /* stylelint-disable-next-line @osd/stylelint/no_restricted_properties */
```

Comment:
not a blocker, but I prefer that all inline lint exceptions have an explanation

Reply:
thanks will PR that right after this is merged.